languages/go/exercises/concept/basic-slices/.docs/hints.md

```diff
@@ -0,0 +1,26 @@
+# Hints
+
+### General
+
+- slices in Go are zero-based. The first index in a slice is `0`.
+
+### 1. Retrieve a card from a stack
+
+- To get the `n`th item from a slice use `slice[n]`.
```

Comment:
This gives away the exact syntax that the student has to use to solve the problem, correct? If so, it might be helpful to name the concept that the student can use (indexing here), and link to a page that shows how to use it. This means that the student won't just be able to copy/paste, which is better for the learning experience, as well as the student learning about the resource (e.g. documentation page) being referred to.